core/templates/domain/question/pretest-question-backend-api.service.spec.ts

```diff
@@ -133,4 +133,19 @@ describe('Pretest question backend API service', function() {
       expect(failHandler).toHaveBeenCalled();
     })
   );
+
+  it('should call successHandler with [] when storyId is empty',
```

Comment:
```suggestion
  it('should call successHandler with an empty array when storyId is empty',
```

Reply:
I added these suggestions through an additional commit. Thank you, this does make it more clear.